Bugsnag/Helpers/BSGWatchKit.h

```diff
@@ -0,0 +1,19 @@
+//
+//  BSGWatchKit.h
+//  Bugsnag
+//
+//  Created by Karl Stenerud on 10.05.22.
+//  Copyright © 2022 Bugsnag Inc. All rights reserved.
+//
+
+#if __has_include(<WatchKit/WatchKit.h>)
+
+#import <WatchKit/WatchKit.h>
+
+#define UIApplicationWillTerminateNotification        @"UIApplicationWillTerminateNotification"
```

Comment:
Just checking - watchOS sends this notification?

Reply:
Yup, verified it on a real device.